docs/developer-docs/latest/update-migration-guides/migration-guides/v4/code/backend/policies.md

```diff
@@ -0,0 +1,125 @@
+description: Migrate policies from Strapi v3.6.8 to Strapi v4.0.x with step-by-step instructions
+canonicalUrl:  Used by Google to index page, should start with https://docs.strapi.io/ — delete this comment when done [paste final URL here]
+---
+
+<!-- TODO: update SEO -->
+
+# v4 code migration: Updating policies
+
+!!!include(developer-docs/latest/update-migration-guides/migration-guides/v4/snippets/code-migration-intro.md)!!!
+
+::: strapi v3/v4 comparison
+
+In both Strapi v3 and v4, policies handle authorization. Policies can be global or scoped (i.e. applied to a specific API or plugin) and can be applied to REST controller's actions or to GraphQL resolvers.
+
+In Strapi v3, policies are Koa middlewares accepting or rejecting requests based on the REST context. As middlewares, v3 policies always receive a Koa context, either coming from the REST request or built from the GraphQL resolver arguments.
+
+In Strapi v4, [policies](/developer-docs/latest/development/backend-customization/policies.md#policies) are functions that should return `true` or `undefined` for the request to be accepted. v4 policies receive a custom context based on where the policy has been called from (e.g. a REST controller’s action or a GraphQL resolver).
+:::
+
+To migrate a policy to Strapi v4:
+
+1. Move the policy file in the appropriate folder, depending on the policy scope:
+  
+    | Policy scope      | Folder                                                                                                                                                                                                           |
+    | ----------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+    | Global            | `./src/policies`                                                                                                                                                                                                 |
+    | A specific API    | `./src/api/<api-name>/policies`                                                                                                                                                                                  |
+    | A specific plugin | `./src/plugins/<plugin-name>/policies`<br/><br/>(see [plugin policies migration](/developer-docs/latest/update-migration-guides/migration-guides/v4/plugin/update-folder-structure.md#moving-policies) documentation) |
+
+2. (_optional_) Update the policy code. Strapi v4 policies are functions returning `true` or `undefined` to authorize the request.
+
+::: note
+In Strapi v4, depending on whether the policy is applied to REST or GraphQL, the function has access to different contexts:
+
+- Both REST and GraphQL contexts have access to the `is(type: string): boolean`, `type: string`, and `state: object` attributes.
```

Comment:
We shouldn't be using typescript references here since we don't support typescript yet.
Same reason we removed them here: https://github.com/strapi/documentation/pull/746